.scripts/build_steps.sh

```diff
@@ -33,9 +33,9 @@ CONDARC
 
 
 mamba install --update-specs --yes --quiet --channel conda-forge \
-    conda-build pip boa conda-forge-ci-setup=3
+    conda-build pip boa conda-forge-ci-setup=3 "py-lief<0.12"
```

Comment:
Why are we adding this `py-lief` restriction?

Reply:
I found https://github.com/conda-forge/conda-forge.github.io/issues/1823#issuecomment-1299040075

Reply:
and https://github.com/conda-forge/conda-smithy/pull/1684/